From e4e383f9e3279bc338f9ee4d0c0ef570463ec656 Mon Sep 17 00:00:00 2001
From: Jean-Michel Trivi <jmtrivi@google.com>
Date: Mon, 30 Sep 2024 15:00:44 -0700
Subject: [PATCH 158/436] Flag for speaker cleanup AudioAttributes usage

Flag: android.media.audio.speaker_cleanup_usage
Bug: 355050846
Test: atest AudioAttributesTest
Change-Id: I9dc028efc099de6ddf6d4a29fd5b28fa3529b40b
---
 media/audio/aconfig/audio_framework.aconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/media/audio/aconfig/audio_framework.aconfig b/media/audio/aconfig/audio_framework.aconfig
index 0b434f7d45..084caf9b92 100644
--- a/media/audio/aconfig/audio_framework.aconfig
+++ b/media/audio/aconfig/audio_framework.aconfig
@@ -163,6 +163,13 @@ flag {
     bug: "367816690"
 }
 
+flag {
+    name: "speaker_cleanup_usage"
+    namespace: "media_audio"
+    description: "Support new AudioAttributes usage for speaker cleanup"
+    bug: "355050846"
+}
+
 # TODO remove
 flag {
     name: "volume_ringer_api_hardening"
-- 
2.43.0

